Source/Relativity.Test.Helpers/Relativity.Tests.Helpers.Tests.Unit/ArtifactHelpers/FieldsTest.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using kCura.Relativity.Client;
+using Moq;
+using NUnit.Framework;
+using Relativity.Test.Helpers;
+using Relativity.Test.Helpers.ArtifactHelpers;
+using Relativity.Test.Helpers.ArtifactHelpers.Interfaces;
+using TestHelpersKepler.Interfaces.TestHelpersModule.v1.Models;
+
+namespace Relativity.Tests.Helpers.Tests.Unit.ArtifactHelpers
+{
+	[TestFixture]
+	public class FieldsTest
+	{
+		private IHttpRequestHelper _httpRequestHelper;
+		private readonly int _workspaceId = 1017834;
+
+		private IFields _fieldsHelper;
```

Comment:
we should rename it to Sut as we have been following this convention for all our tests.